files/rd_job_diff.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
+
+project="$1"
+job="$2"
+path="$3"
+format="$4"
+
+rd jobs list -p "$project" -J "$job" -f "/tmp/$job.$format" -F $format
```

Comment:
Can you please use mktemp to create a tmpfile and use that? That ensures the paths don't use anything predictable.

Reply:
Updated the script